internal/pkg/service/cli/cmd/remote/create/table/dialog.go

```diff
@@ -30,7 +30,15 @@ type columnsDefinitionMethod int
 func AskCreateTable(args []string, branchKey keboola.BranchKey, allBuckets []*keboola.Bucket, d *dialog.Dialogs, f Flags, backends []string) (table.Options, error) {
 	opts := table.Options{}
 
-	if f.OptionsFrom.IsSet() && f.ColumnsFrom.IsSet() && slices.Contains(backends, BackendBigQuery) {
+	if f.OptionsFrom.Value != "" {
+		if !f.ColumnsFrom.IsSet() {
+			return opts, errors.Errorf("columns-from must be set for bigquery settings")
+		}
+
+		if !slices.Contains(backends, BackendBigQuery) {
+			return opts, errors.Errorf("backend have to be `bigquery`")
+		}
```

Comment:
`"backend have to be `bigquery`"` why?

`"backend have to be `bigquery` to use --options-from"

---------

I prefer to swap the conditions, if the backend is not `bigquery` other checks have no sense.

Reply:
And please use the constant:

```
fmt.Sprintf(`backend have to be "bigquery" to use --options-from flag`)
```